starter-code/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title> title </title>
+    <link rel='stylesheet' href='public/stylesheets/style.css' />
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>    
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+  </head>
+  <body>
+    <h1>axios</h1>
+    <p>Welcome to axios</p>
+
+<script> 
+
+</script> 
+  </body>
+  <script src="./public/javascripts/financial-data.js" charset="utf-8"></script>
```

Comment:
OJO al poner las rutas, lo suyo hubiera sido `src="public/javascripts/financial-data.js"` ,  aunque funcione como lo habéis puesto, cuando se ejecute en otro sitio que no sea local, podría no ir. Tened esto en cuenta para futuros desarrollos ;)